### PR TITLE
Add credits field and fix spacing according to style guide

### PR DESCRIPTION
**Pull request by maceonthompson**
Also adds a getter function for the credits field.

### CODE REVIEW

File: src/main/java/com/google/sps/data/Course.java [hunk 1]
```diff
@@ -20,42 +20,43 @@
...
+  }
+
+  public boolean getIsRequired() {
+    return this.isRequired;
+  }
+
+  public List<Section> getSections() {
+    return this.sections;
+  }
 }
```
Comment (achillesecos) [inline src/main/java/com/google/sps/data/Course.java]: Add newline here

Comment (maceonthompson) [reply src/main/java/com/google/sps/data/Course.java]: Resolvd!

**Review by achillesecos — APPROVED** [review]
LGTM

File: src/main/java/com/google/sps/data/Course.java [hunk 2]
```diff
@@ -20,42 +20,43 @@
...
+
+  public String getSubject() {
+    return this.subject;
+  }
+
+  public float getCredits() {
+    return this.credits;
+  }
+
+  public boolean getIsRequired() {
```
Comment (kristen-law) [inline src/main/java/com/google/sps/data/Course.java]: Getters for booleans generally drop the `get` so it reads easier: `isRequired()`

**Review by kristen-law — CHANGES_REQUESTED** [review]

**Review by kristen-law — APPROVED** [review]

**Pull request merged**